masterlist.yaml

```diff
@@ -136,3 +136,75 @@ plugins:
     url: [ 'http://www.nexusmods.com/fallout4/mods/2142' ]
     after:
       - 'Simple Bug Fixes.esp'
+
+  - name: 'DarkerNights-TrueStorms-LITE-Audio.esp'
+    url: [ 'http://www.nexusmods.com/fallout4/mods/191' ]
+    after:
+      # Place DarkerNights-TrueStorms-LITE-Audio.esp after TrueStormsFO4-LITE-Audio.esp
+      # to ensure the magic of Darker Nights is applied to WTHR records modified by the True Storms LITE Audio addon
+      - 'TrueStormsFO4-LITE-Audio.esp'
+    msg:
+      # Display a warning when Darker Nights patch for True Storms LITE Audio addon is NOT used along with the True Storms LITE Audio addon
```

Comment:
This is a superflous comment, it just repeats exactly what is spelled out in the message below.